website/docs/features/onnx/about.md

```diff
@@ -43,6 +75,30 @@ SynapseML now includes a Spark transformer to bring an trained ONNX model to Apa
 
 4. Call `transform` method to run inference on the input DataFrame.
 
+## Model Slicing
+If you wish to observe or use intermediate nodes of a model, you can slice the model at known outputs, keeping only parts of
+the model that are needed for those nodes. This will generate a new model that you can save and/or use to transform.
+
+This feature is used under the covers by the ImageFeaturizer, which uses ONNX models. The OnnxHub manifest entry for each model includes which intermediate nodes
```

Comment:
Might want to choose something a bit more formal than under the covers. Also if you run this through acrolinx it might help spot these mannerisms